build.gradle

```diff
@@ -6,14 +6,15 @@ plugins {
     id 'com.github.ben-manes.versions' version '0.47.0'
     id 'com.gorylenko.gradle-git-properties' version '2.4.1'
     id 'io.spring.dependency-management' version '1.1.0'
-    id 'net.serenity-bdd.serenity-gradle-plugin' version '3.7.0'
-    id 'org.flywaydb.flyway' version '9.20.0'
+    id 'net.serenity-bdd.serenity-gradle-plugin' version '3.9.0'
+    id 'org.flywaydb.flyway' version '9.21.0'
     id 'org.owasp.dependencycheck' version '8.3.1'
-    id 'org.sonarqube' version '4.2.1.3168'
-    id 'org.springframework.boot' version '3.1.1'
+    id 'org.sonarqube' version '4.3.0.3225'
+    id 'org.springframework.boot' version '3.1.2'
```

Comment:
I'd suggest against doing these lib updates and better do a rebase on master that should bring your branch up to date (though that rebase might be a bit painful)